src/Symfony/Bridge/Twig/composer.json

```diff
@@ -21,22 +21,22 @@
     },
     "require-dev": {
         "fig/link-util": "^1.0",
-        "symfony/asset": "~2.8|~3.0",
-        "symfony/finder": "~2.8|~3.0",
-        "symfony/form": "^3.2.7",
-        "symfony/http-kernel": "~3.2",
+        "symfony/asset": "~2.8|~3.0|~4.0",
+        "symfony/finder": "~2.8|~3.0|~4.0",
+        "symfony/form": "^3.2.7|~4.0",
+        "symfony/http-kernel": "~3.2|~4.0",
         "symfony/polyfill-intl-icu": "~1.0",
-        "symfony/routing": "~2.8|~3.0",
-        "symfony/templating": "~2.8|~3.0",
-        "symfony/translation": "~2.8|~3.0",
-        "symfony/yaml": "~2.8|~3.0",
-        "symfony/security": "~2.8|~3.0",
-        "symfony/security-acl": "~2.8|~3.0",
-        "symfony/stopwatch": "~2.8|~3.0",
-        "symfony/console": "~2.8|~3.0",
-        "symfony/var-dumper": "~2.8.10|~3.1.4|~3.2",
-        "symfony/expression-language": "~2.8|~3.0",
-        "symfony/web-link": "~3.3"
+        "symfony/routing": "~2.8|~3.0|~4.0",
+        "symfony/templating": "~2.8|~3.0|~4.0",
+        "symfony/translation": "~2.8|~3.0|~4.0",
+        "symfony/yaml": "~2.8|~3.0|~4.0",
+        "symfony/security": "~2.8|~3.0|~4.0",
+        "symfony/security-acl": "~2.8|~3.0|~4.0",
```

Comment:
Are we sure that we will bump the release for the Security Acl component? I don't think so (we did not really add anything there).